opensrp-chw-anc/src/main/java/org/smartregister/chw/anc/activity/BaseAncMemberProfileActivity.java

```diff
@@ -268,7 +268,7 @@ public void openFamilyDueServices() {
 
     @Override
     public void setProfileImage(String baseEntityId, String entityType) {
-        imageRenderHelper.refreshProfileImage(baseEntityId, imageView, NCUtils.getMemberProfileImageResourceIDentifier(entityType));
+        imageRenderHelper.refreshProfileImage(baseEntityId, imageView, R.drawable.anc_woman);
```

Comment:
Is there a need for this to be returned by a util function that returns the ANC/PNC resource identifier?

Reply:
Changed this for consistency